webapp/lib/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "pyroscope",
+  "version": "0.0.1-%VERSION%",
+  "main": "index.lib.js",
+  "browser": "index.js",
+  "license": "MIT"
```

Comment:
Not sure but I think we should probably list dependencies here.

Reply:
That's a great question. For the browser there's no need to install anything, since we package everything.

Now for node it's a different story, but I don't know how to separate those. Maybe we need different package.json?